setup.py

```diff
@@ -69,10 +69,6 @@
 
 from _vendor.packaging.version import Version
 
-# Get geos_version from GEOS dynamic library, which depends on
-# GEOS_LIBRARY_PATH and/or GEOS_CONFIG environment variables
-from shapely._buildcfg import geos_version_string, geos_version, \
-        geos_config, get_geos_config
```

Comment:
All those imported variables are either not used in this file or either redefined a bit lower